keyboards/bbrfkr/dynamis/dynamis.c

```diff
@@ -16,18 +16,18 @@
 
 #include "dynamis.h"
 
-#ifdef ENCODER_ENABLE
+#ifndef VIA_ENABLE
```

Comment:
This should still be reverted. If you use the `encoder_update_user` function in the via keymap, and return false, it will prevent this code from being executed (that's what that first "if" is doing).  

Also, you wan to wrap the function in the encoder ifdef, so that if someone disables the feature, it doesn't error out during compiling. 
```suggestion
#ifdef ENCODER_ENABLE
```

Reply:
Thanks. I fixed it.